src/uct/ib/dc/dc_mlx5.c

```diff
@@ -73,6 +74,16 @@ ucs_config_field_t uct_dc_mlx5_iface_config_sub_table[] = {
      ucs_offsetof(uct_dc_mlx5_iface_config_t, tx_policy),
      UCS_CONFIG_TYPE_ENUM(uct_dc_tx_policy_names)},
 
+    {"LAG_PORT_AFFINITY", "auto",
+     "Specifies how DCI select port. The values are:\n"
+     " auto     Auto detect the lag mode info.\n"
+     "          For lag queue affinity mode, config port affinity.\n"
+     "          For other lag mode, do not config port affinity.\n"
+     " on       Config port affinity under lag mode.\n"
+     " off      Do not config port affinity under lag mode.\n",
```

Comment:
IIUC it should be something like bellow:
"on - Force port affinity", BTW, if it's not supported IMO we should fail or print error/warning
"off - Use default port affinity policy"
"auto" looks redundant

Reply:
@evgeny-leksikov
>
> "on - Force port affinity", BTW, if it's not supported, IMO we should fail or print error/warning
>
This only effective for LAG device. For single port device, it won't be effective. Should we print error/warning for the case that UCX run on single port device?

>
> "auto" looks redundant
>
"auto" could let UCX to check the lag mode for DC and decide whether it needs to set affinity (with this PR, set DCI affinity if it's queue_affinity lag mode, or don't config the DCI affinity)

>
> "off - Use default port affinity policy"
>
If it does in this way, it looks strange. "off" or "no" means don't config DCI affinity.

Reply:
@evgeny-leksikov All other comments are fixed except below one, please help check it.
https://github.com/openucx/ucx/pull/8382#discussion_r945733418

Reply:
@evgeny-leksikov
>
>1. it should be documented
> 2. if user does smth wrong we should warn/err/fail
>
Done

>
> what does mean "don't config DCI affinity"? and what's the difference between off and auto?
>
1. ```don't config DCI affinity``` means that it won't call [uct_ib_mlx5_devx_set_qpc_port_affinity](https://github.com/openucx/ucx/blob/v1.13.0/src/uct/ib/mlx5/dv/ib_mlx5_dv.c) for DCI (do not bind DCI with specific port)
2. ```off``` means that it won't call [uct_ib_mlx5_devx_set_qpc_port_affinity](https://github.com/openucx/ucx/blob/v1.13.0/src/uct/ib/mlx5/dv/ib_mlx5_dv.c) for DCI whatever the lag mode is.
3. ```auto``` means that
1) If the lag mode is ```QUEUE_AFFINITY```, set DCI affinity (bind DCI with specific port)
2) If the lag mode is ```PORT_SELECT_FT```, do not set DCI affinity (do not bind DCI with specific port)

Reply:
I don't think it's clear for UCX user, can it be described somehow simpler with focus on result behaviour?

Reply:
@evgeny-leksikov I've refined the documentation. Please help check it.